### PR TITLE
[MRG] DOC Adding contributing page in doc

### LINKED ISSUE #962: Add a pointer to dev docs in the README.rst
We should add a pointer to the developer guidelines in the README.rst so that a new developer can get up to speed/

### PR DESCRIPTION
**Pull request by aabadie**
This should fix #962 and #891 

I tried to reused documentation from external sources. The contributing file is at the root of the project so that GitHub propose a link a it each time one adds an issue.

### CODE REVIEW

**Comment by aabadie** [comment]
@AlexandreAbraham, it would be great to have in next release :)

**Comment by AlexandreAbraham** [comment]
+1 for including it in the upcoming release. This is a good start for the contributing part. Let's wait for CircleCI.

PS: have you seen that the appveyor build has been cancelled... Does anybody know the reason?

**Comment by lesteve** [comment]
Could it be because of this:

```
More recent "master" branch build has been stated.
```

**Comment by aabadie** [comment]
@AlexandreAbraham, @lesteve, all tests passed, should be ready to be merged :)

File: doc/install_doc_component.html [hunk 1]
```diff
@@ -46,10 +46,10 @@
       <p><strong>We recommend that you install a complete scientific Python
        distribution like 64 bit <a class="reference external" href="https://
        store.continuum.io/cshop/anaconda/" target="_blank">Anaconda</a>
-       </strong>. Since it meets all the requirements of nilearn, it will save 
+       </strong>. Since it meets all the requirements of nilearn, it will save
        you time and trouble. You could also check <a class="reference external"
-       href="http://code.google.com/p/pythonxy/" target="_blank">PythonXY</a>
        as an alternative.</p>
+       href="http://code.google.com/p/pythonxy/" target="_blank">PythonXY</a>
```
Comment (KamalakerDadi) [inline doc/install_doc_component.html]: May be its not part of your PR but the link is outdated here.

You could replace with http://python-xy.github.io/ if its the suitable one. Apart from that fine with me.

Comment (aabadie) [inline doc/install_doc_component.html]: Indeed, it's not part of the "real" changes of this PR but a misconfiguration of my IDE automatically removed trailing spaces in html files. The broken link fix could be part of another PR I think.

File: README.rst [hunk 2]
```diff
@@ -66,16 +66,7 @@ http://nilearn.github.io/introduction.html#installation.
 Development
 ===========
 
-Code
-----
+The project is hosted on https://github.com/nilearn/nilearn
```
Comment (lesteve) [inline README.rst]: This README.rst will be mostly looked at on github, not sure that this line is worth it.

Then just say "Detailed instructions ..."

File: doc/install_doc_component.html (hunk 1 above)
Comment (lesteve) [inline doc/install_doc_component.html]: For the record the link is not broken per se, it's just a "This project has moved" page with the up-to-date link to github.io.

File: README.rst (hunk 2 above)
Comment (aabadie) [inline README.rst]: I created #980 to fix this.
